src/test/DittoMachine.t.sol

```diff
@@ -73,7 +73,7 @@ contract ContractTest is TestBase {
         vm.startPrank(address(bidder));
         currency.approve(dmAddr, MIN_AMOUNT_FOR_NEW_CLONE);
 
-        (uint cloneId, uint protoId) = dm.duplicate(nftAddr, FLOOR_ID, currencyAddr, MIN_AMOUNT_FOR_NEW_CLONE, true, 0);
+        (uint cloneId, uint protoId) = dm.duplicate(address(bidder), nftAddr, FLOOR_ID, currencyAddr, MIN_AMOUNT_FOR_NEW_CLONE, true, 0);
```

Comment:
@calvbore shouldn't this revert? `bidder` doesn't implement the erc721 receiver.

Reply:
nvm, this is minting a clone, that's why